packages/components/table/src/table/style-helper.ts

```diff
@@ -109,7 +108,7 @@ function useStyle<T>(
   onMounted(async () => {
     setScrollClass('is-scrolling-left')
     store.updateColumns()
-    await nextTick()
+    // await nextTick()
```

Comment:
Remove it or reserve it. Don't use comment.


/cc @msidolphin Should we really remove it?